libs/mock-block-dock/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@local/tsconfig/dom+jsx.json",
-  "include": ["assets.d.ts", "src", "dev"]
+  "include": ["assets.d.ts", "src", "dev"],
+  "compilerOptions": {
+    "skipLibCheck": true
+  }
```

Comment:
`echarts` has a bunch of type errors which this allows us to ignore. I don't know why they weren't happening before or if there's a better way of fixing this rather than effectively ignoring the problem (and typings in every other library).